### PR TITLE
Bundle name and namespace

### PR DESCRIPTION
**Pull request by hmert**
This pull requeset changes bundle's name and namespace.

### CODE REVIEW

File: Helper/AdvancedCitationHelper.php [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <?php
 
-namespace BulutYazilim\AdvancedCitationBundle\Helper;
+namespace OJS\AdvancedCitationBundle\Helper;
```
Comment (utkuaydin) [inline Helper/AdvancedCitationHelper.php]: This one is all-uppercase while others aren't.

File: Helper/AdvancedCitationHelper.php [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 use Doctrine\Common\Persistence\ObjectManager;
 use GuzzleHttp\Client;
 use Ojs\JournalBundle\Entity\Article;
 use Ojs\JournalBundle\Entity\Citation;
-use BulutYazilim\AdvancedCitationBundle\Entity\AdvancedCitation;
+use OJS\AdvancedCitationBundle\Entity\AdvancedCitation;
```
Comment (utkuaydin) [inline Helper/AdvancedCitationHelper.php]: This one is all-uppercase as well.

**Review by utkuaydin — COMMENTED** [review]
I see no problem other than these two. I will take care of them myself and merge this after testing manually.

File: Helper/AdvancedCitationHelper.php (hunk 2 above)
Comment (hmert) [reply Helper/AdvancedCitationHelper.php]: done

File: Helper/AdvancedCitationHelper.php (hunk 1 above)
Comment (hmert) [reply Helper/AdvancedCitationHelper.php]: done

**Pull request merged**